components/login-form.html

```diff
@@ -0,0 +1,41 @@
+<div class="flex justify-center items-center h-screen bg-gray-200 px-6">
+    <div class="p-6 max-w-sm w-full bg-white shadow-md rounded-md">
```

Comment:
Just some small tweaks to the mobile view. That together with the suggestion above will result in the screenshot below.

```suggestion
    <div class="p-4 sm:p-6 max-w-sm w-full bg-white shadow-md rounded-md">
```

![Screen Shot of login form at full width on mobile](https://user-images.githubusercontent.com/2221746/94313516-27177680-ff7f-11ea-9f1e-55bd2430600b.png)